src/Emogrifier.php

```diff
@@ -1106,6 +1093,32 @@ private function attributeValueIsImportant($attributeValue)
         return strtolower(substr(trim($attributeValue), -10)) === '!important';
     }
 
+    /**
+     * Copies $cssRule into the style attribute of $node.
+     *
+     * Note: This method does not check whether $cssRule matches $node.
+     *
+     * @param \DOMElement $node
+     * @param string[][] $cssRule
+     *
+     * @return void
+     */
+    private function copyInlineableCssToStyleNode(\DOMElement $node, array $cssRule)
```

Comment:
I am not that sure of the name of the method. Maybe `copyInlineableCssToStyleAttribute` or `copyInlineableCssToNodeStyleAttribute` would be better? What do you think?

Reply:
Oh, of course. Good catch! The "style node" does not make sense here.

I'll change it to `copyInlineableCssToStyleAttribute`.